corehq/apps/users/static/users/js/roles.js

```diff
@@ -359,8 +359,22 @@ hqDefine('users/js/roles',[
                 };
             },
         };
-        self.submitNewRole = function () {
-            // moved saveOptions inline
+        self.roleError = ko.observable("");
+        self.setRoleError = function (form, error) {
+            self.roleError(error);
+            $(form).find('.modal-footer').addClass('has-error');
+            $(form).find('[type="submit"]').enableButton();
+        };
+        self.clearRoleError = function (form) {
+            self.roleError("");
+            $(form).find('.modal-footer').removeClass('has-error');
+        };
+        self.clearRoleForm = function (_, event) {
```

Comment:
I don't think `_` is a good dummy variable name here, since there's already a module-level `_` for underscore.

Reply:
c5a7d0e9935